lib/src/chart/bar_chart/bar_chart_painter.dart

```diff
@@ -668,6 +672,10 @@ class BarChartPainter extends AxisChartPainter<BarChartData> {
         bottomLeft: radius,
         bottomRight: radius);
     _bgTouchTooltipPaint.color = tooltipData.tooltipBgColor;
+    if (tooltipData.tooltipBorderWidth > 0) {
```

Comment:
Can you please check the opacity of color too here?
Because sometimes people set a transparent color, which we don't need to draw anything.
(Maybe it's better to have an extension function like `borderSide.isVisible()`)

```dart
extension BorderSideExtension on BorderSide {
  bool isVisible() => color.opacity > 0 && width > 0;
}
```

Reply:
Please apply it to other places.